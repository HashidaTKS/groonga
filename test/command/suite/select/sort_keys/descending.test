table_create Numbers TABLE_NO_KEY
column_create Numbers value COLUMN_SCALAR Int32

load --table Numbers
[
{"value": 3},
{"value": -1},
{"value": 2}
]

select Numbers --sort_keys -value
