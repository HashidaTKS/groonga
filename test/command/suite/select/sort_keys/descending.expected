table_create Numbers TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Numbers value COLUMN_SCALAR Int32
[[0,0.0,0.0],true]
load --table Numbers
[
{"value": 3},
{"value": -1},
{"value": 2}
]
[[0,0.0,0.0],3]
select Numbers --sort_keys -value
[[0,0.0,0.0],[[[3],[["_id","UInt32"],["value","Int32"]],[1,3],[3,2],[2,-1]]]]
